src/aspire/source/image.py

```diff
@@ -584,7 +584,6 @@ def save(
         logger.info("save metadata into STAR file")
         filename_indices = self.save_metadata(
             starfile_filepath,
-            new_mrcs=True,
```

Comment:
I'm not totally convinced we always want to (re)write the mrc images.  Maybe I am missing something.